app/controllers/movies_controller.rb

```diff
@@ -1,5 +1,24 @@
 class MoviesController < ApplicationController
   def index
-    @movies = Movie.all
+    @movies = []
+    if params[:search] == 'top_movies'
+      conn = Faraday.new('https://api.themoviedb.org')
+      response1 = conn.get("/3/movie/popular?api_key=#{Figaro.env.movie_api_key}&page=1")
+      response2 = conn.get("/3/movie/popular?api_key=#{Figaro.env.movie_api_key}&page=2")
+      results1 = JSON.parse(response1.body, symbolize_names: true)
+      results2 = JSON.parse(response2.body, symbolize_names: true)
+      forty_movies = []
+      @movies = forty_movies.push(results1[:results],results2[:results]).flatten
+    elsif params[:commit] == 'Find Movie'
+      conn = Faraday.new('https://api.themoviedb.org')
+      response1 = conn.get("/3/search/movie?api_key=#{Figaro.env.movie_api_key}&query=#{params[:movie][:search]}&page=1&include_adult=true")
+      response2 = conn.get("/3/search/movie?api_key=#{Figaro.env.movie_api_key}&query=#{params[:movie][:search]}&page=2&include_adult=true")
```

Comment:
let's change Figaro.env.movie_api_key to ENV['movie_api_key'] per today's lesson

Reply:
Agree